jaeger-core/src/main/java/com/uber/jaeger/Constants.java

```diff
@@ -42,4 +42,19 @@
    * root span, so that the trace can be found in the UI using this value as a correlation ID.
    */
   public static final String DEBUG_ID_HEADER_KEY = "jaeger-debug-id";
+
+  /**
+   * The name of the tag used to report client version.
+   */
+  public static final String JAEGER_CLIENT_VERSION_TAG_KEY = "jaeger.version";
+
+  /**
+   * The name used to report host name of the process.
+   */
+  public static final String TRACER_HOSTNAME_TAG_KEY = "jaeger.hostname";
```

Comment:
sure. Although I don't know why you need a backend change. The idea here is that when people use Jaeger backend, they should be using jaeger.thrift reporters (udp or http). Only when they use Zipkin backend they would use zipkin.thrift reporter, where we convert tracer tags into first-in-process span tags.